k8s-operator/api-proxy/proxy.go

```diff
@@ -143,7 +143,7 @@ func (ap *APIServerProxy) Run(ctx context.Context) error {
 		}
 	} else {
 		var err error
-		tsLn, err = ap.ts.Listen("tcp", ":80")
+		proxyLn, err = net.Listen("tcp", "localhost:80")
```

Comment:
Does this listener now need closing since we're making it via `net` and not `ap.ts`?

Reply:
I had to double check this one, but the serve method closes the listener for us so I think this is fine as-is